lms/djangoapps/courseware/tests/test_tabs.py

```diff
@@ -155,16 +154,16 @@ def check_can_display_results(
     ):
         """Checks can display results for various users"""
         if for_staff_only:
-            user = self.create_mock_user(is_authenticated=True, is_staff=True, is_enrolled=True)
+            user = self.create_mock_user(is_staff=True, is_enrolled=True)
             self.assertEquals(expected_value, self.is_tab_enabled(tab, self.course, user))
         if for_authenticated_users_only:
-            user = self.create_mock_user(is_authenticated=True, is_staff=False, is_enrolled=False)
+            user = self.create_mock_user(is_staff=False, is_enrolled=False)
             self.assertEquals(expected_value, self.is_tab_enabled(tab, self.course, user))
         if not for_staff_only and not for_authenticated_users_only and not for_enrolled_users_only:
-            user = self.create_mock_user(is_authenticated=False, is_staff=False, is_enrolled=False)
+            user = self.create_mock_user(is_staff=False, is_enrolled=False)
```

Comment:
I think we want to preserve this user not being authenticated, to cover all the possible cases.